virttest/utils_misc.py

```diff
@@ -2565,11 +2565,12 @@ def __get_unit_index(M):
     from_index = __get_unit_index(unit)
     to_index = __get_unit_index(order_magnitude)
     scale = int(factor) ** (to_index - from_index)
-    if scale > 0:
-        data_size = float(value) / abs(scale)
+    data_size = float(value) / scale
+    # Control precision to avoid scientific notaion
+    if data_size.is_integer():
```

Comment:
And it's not a good idea to update output of this function, because it used by many test, if updated is indispensable, please ensure your update not impact other test.

Thanks,
Xu